lua/nvim-tree/actions/node/file-popup.lua

```diff
@@ -32,6 +32,7 @@ local function setup_window(node)
     width = max_width + 1,
     height = #lines,
     noautocmd = true,
+    zindex = 60,
```

Comment:
:+1: 

The default value for floats are 50. In general, values below 100 are recommended, unless there is a good reason to overshadow builtin elements.